### PR TITLE
Feature/2221 privacy policy

### PR DESCRIPTION
**Pull request by ShazSeatter**
Update privacy policy page content so it can be used across several services. 

Issue: https://github.com/Scottish-Natural-Heritage/Licensing/issues/2221

### CODE REVIEW

**Review by paulcoxsnh — APPROVED** [review]
Looks good to me :)

File: src/views/privacy-policy.njk [hunk 1]
```diff
@@ -7,13 +7,21 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
-      <h1 class="govuk-heading-l">Privacy Policy for Gulls (Public Health and Safety)</h1>
+      <h1 class="govuk-heading-l">Privacy Policy for online licensing services</h1>
 
       <p class="govuk-body">This policy applies to the content of the online applications available at:</p>
 
-      <p class="govuk-body"><a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety/">https://licensing.nature.scot/gulls-health-and-safety/</a></p>
+      <p class="govuk-body">Trap registrations: <a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety/">https://licensing.nature.scot/trap-registration/start</a></p>
```
Comment (finchie) [inline src/views/privacy-policy.njk:14]: I think the `href` URL may not be correct here.

File: src/views/privacy-policy.njk [hunk 2]
```diff
@@ -7,13 +7,21 @@
...
+      <p class="govuk-body">Standard forestry operations: <a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety-return/">https://licensing.nature.scot/standard-forestry-operations/start</a></p>
+
+      <p class="govuk-body">Standard forestry operations (returns):<a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety/">https://licensing.nature.scot/standard-forestry-operations-return/start</a></p>
+
+      <p class="govuk-body">Gulls health and safety: <a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety-return/">https://licensing.nature.scot/gulls-health-and-safety/intro</a></p>
+
+      <p class="govuk-body">Gulls health and safety (returns): <a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety-return/">https://licensing.nature.scot/gulls-health-and-safety-return/intro</a></p>
+
+      <p class="govuk-body">Land on which red grouse may be killed or taken: <a class="govuk-link" href="https://licensing.nature.scot/gulls-health-and-safety-return/">https://licensing.nature.scot/red-grouse/apply</a></p>
```
Comment (finchie) [inline src/views/privacy-policy.njk:24]: I think the `href` URLs may not be correct here.

**Review by finchie — COMMENTED** [review]
@ShazSeatter @paulcoxsnh 

Forgive me for butting in on a closed PR - but I spotted some erroneous URLs used in links to Licensing services to which this privacy policy pertains.

**Comment by ShazSeatter** [comment]
Thanks @finchie for spotting that! 

I'll make those changes as well, thanks :)

**Pull request merged**